h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/rabit/RabitTrackerH2O.java

```diff
@@ -42,27 +58,11 @@ public RabitTrackerH2O(int workers) throws IOException {
             } catch (java.io.IOException e) {
                 this.port++;
                 if(this.port > 9999) {
-                    throw e;
+                    throw new RuntimeException("Failed to bind Rabit tracker to a socket in range 9091-9999", e);
```

Comment:
Fixed port range?

Reply:
Not ideal however fine for now - can be parametrized later

Reply:
@mmalohlava yes I was talking about this before - we can parametrize this without much problem *but* this still won't solve the whole problem - Rabit internally opens sockets for C++ which are hardcoded there and we cannot change them unless we change Rabit code...